bin/gen-credits.dart

```diff
@@ -6,8 +6,11 @@ import 'package:http/http.dart';
 import 'package:html/parser.dart';
 import 'package:pub_api_client/pub_api_client.dart';
 import 'package:pubspec_parse/pubspec_parse.dart';
+import 'package:spotube/models/logger.dart';
 
 void main() async {
+  final logger = getLogger("GenCredits");
```

Comment:
Please revert this file's changes. It's not for debug purposes. The output needs not to be decorated